ElementX/Sources/FlowCoordinators/AppLockFlowCoordinator.swift

```diff
@@ -71,6 +76,8 @@ class AppLockFlowCoordinator: CoordinatorProtocol {
     // MARK: - App unlock
     
     private func applicationDidEnterBackground() {
+        unlockTask?.cancel()
```

Comment:
```suggestion
        unlockTask = nil
```
If you decide to use `@CancellableTask`